lib/src/parser.c

```diff
@@ -477,10 +512,10 @@ static Subtree ts_parser__lex(
       self->language
     );
 
-    LOG(
-      "lexed_lookahead sym:%s, size:%u, character:'%c'",
+    LOG_LOOKAHEAD(
       SYM_NAME(ts_subtree_symbol(result)),
       ts_subtree_total_size(result).bytes,
+      "character:'%c'",
```

Comment:
I don't think this `character` part needs to be logged; I think I left it in by accident. It looks like, if you removed this, you could make the macro simpler (no varargs).

Reply:
I don't remember that this was useful for me, so I completely agree with you. Let's make it simpler, I've pushed additional commit. Also added a backslash escaping to make it clear what it is in case of \\\n.